providers/ethereum-provider/src/index.ts

```diff
@@ -64,27 +65,21 @@ export interface EthereumProviderOptions {
 
 class EthereumProvider implements IEthereumProvider {
   public events: any = new EventEmitter();
-
-  private rpc: EthereumRpcConfig;
-
+  public rpc: EthereumRpcConfig;
```

Comment:
Is there a reason why `this.rpc` was rescoped from `private` to `public`? Couldn't find an example of it being being called from outside the class anywhere else in the diff

Reply:
I think we did this with pedro during quick fix sesh, but I just changed it back to private, ran tests and checked ts, all good. Nice catch 👌